src/MICore/MICoreResources.resx

```diff
@@ -154,7 +154,7 @@ Error: {1}</value>
     <value>Launch options string provided by the project system is invalid. {0}</value>
   </data>
   <data name="Error_InvalidLocalExePath" xml:space="preserve">
-    <value>Invalid path to executable file path '{0}'. File must be a valid file name that exists on the Visual Studio computer.</value>
+    <value>Invalid path to executable file path '{0}'. File must be a valid file name that exists on the computer.</value>
```

Comment:
This string is only used for core dumps anymore. So I would suggest just --
Invalid path to executable file path '{0}'. File must be a valid file name that exists.